src/layer/place.js

```diff
@@ -146,9 +143,6 @@ export const town = {
   minzoom: 4,
   maxzoom: 13,
   "source-layer": "place",
-  metadata: {
-    "americana:text-field-localized": true,
-  },
```

Comment:
So far, I’ve limited the glosses to cities to avoid clutter at mid zoom levels. This is already more generous than print atlases, which limit the glosses to “world-class” cities with well-known English names. However, we could explore factoring out the giant expression below and reusing it on the `place=town` and `place=village` layers at much higher zoom levels. It might require some fine-tuning to maintain the desired label density. The Wikidata items on these smaller settlements are less likely to have gotten cleaned up since an import, so this would be a nice way to surface data in need of attention. I’m thinking we could save that for tail work once we’ve proven out glosses in the city layer.